internal/service/systemd.go

```diff
@@ -133,8 +139,34 @@ func NewSystemd(name string, serviceNamePrefix string, units map[string]string)
 	return NewSystemdWithSuffix(name, serviceNamePrefix, PodSuffix, ServiceSuffix, units)
 }
 
+func newDbusConnection() (*dbus.Conn, error) {
+	if os.Getenv("XDG_RUNTIME_DIR") != "" && os.Getenv("XDG_RUNTIME_DIR") != "/run" {
```

Comment:
`os.Getenv("XDG_RUNTIME_DIR")` is called 3 times in this context, can it be stored in a local variable?

Reply:
Yes, thanks.